server.js

```diff
@@ -24,12 +80,36 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+// Home page with listed endpoints
+app.get('/', (req, res) => {
+	res.send(listEndpoints(app));
```

Comment:
Again, thank you for the idea. I think I missed this part somewhere from our tutorials.